doc/ref_cert/lfn/chapters/chapter05.md

```diff
@@ -1,21 +1,184 @@
-[<< Back](../)
-
-# 5. VNF E2E C&V Framework Requirements
-<p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
-
-## Table of Contents
-* [5.1 Introduction](#5.1)
-* [5.2 Heading](#5.2)
-* [5.3 Heading](#5.3)
-
-<a name="5.1"></a>
-## 5.1 Introduction
-
-> Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
-
-<a name="5.2"></a>
-## 5.2 Heading
-
-
-<a name="5.3"></a>
-## 5.3 Heading
+\<\< Back
+
+# 5. VNF E2E C&V Framework Requirements
+
+## Table of Contents
+
+* [5.1 Introduction](#5.1)
+* [5.2 Scope](#5.2)
+* [5.3 Platform Requirements](#5.3)
+* [5.4 Certification Process](#5.4)
+
+<a name="5.1"></a>
+### 5.1 Introduction
+
+As Telcom industry is evolving along with virtualization and software based
+cloud network function from hardware based network function, and CNTT is
+defining the required architecture and model for NFVi along with VNF
+specification. As deferent vendors and open-source produce network functions,
+it’s important to define and certify these network functions based on the CNTT
+defined VNF specification. In this chapter the scope and requirements is defined
+as reference for VNF certification platform, which helps to perform the various
+compliance and verification (C&V) testing and submit results to LFN OVP
+certification portal.
+
+<a name="5.2"></a>
+### 5.2 Scope
+
+The requirements defined in next section for the platform would be scoped with
+guidelines defined as below to support VNF compliance & verification
+certification testing.
+
+![](media/c665a3d13461f67ea8729042cf8d975d.png)
+
+#### Standards/Profiles
+
+-   ETSI (TOSCA)
+
+-   GSMA
+
+-   ONAP VNFREQS (HOT)
+
+#### Test cases
+
+Refer *chapter RC-06* for more details on test case requirements defined for VNF
+under CNTT. Platform should support to managed and execute these test cases.
+
+NOTE: For certification, only compliance and verification test cases will be
+considered, but in future, it could be extent to validation and Performance
+related testing.
+
+##### Compliance
+
+Perform compliance check based on
+
+-   TOSCA using ETSI SOL004 & SOL001

```

Comment:
TOSCA and HEAT are out of scope of CNTT. it might worth having a quick (including @mf4716 to discuss this)

Reply:
Expectation is to perform Heat based VVP/VTP validations for VNF testing.  This confirms ICE compliance before the VNF (golden, actual, protoype) can be instantiated and removed.  First step is compliance.  Do you @rabi-abdel or @mkr1481 see this differently?

Reply:
we shouldn't limit the VNF template, both Tosca and Heat should be considered, because these VNFs can all run on  CNTT compliant NFVI, as a common NFVI, it should carry VNF defined by various modules, otherwise it is not a general and common NFVI

Reply:
agree @yangyanyj - @rabi-abdel , can we resolve conversation?

Reply:
I think the whole compliance part needs to be removed. CNTT will scope will not include compliance to TOSCA or HEAT (this is MANO issue not CNTT). my suggestion is to remove compliance section, approve and merge, then we can discuss it further.

Reply:
@rabi-abdel , the goals were to perform VVP as part of HEAT validation for onboarding / instantiating VNFs.  This would allow for validation that the VNF meets specs outlined by the NFVI.  Agree, out of scope are VNF functional testing, and TOSCA validation, but HEAT via VVP execution, which can be run separately from ONAP, is in scope.  Please contact me if live conversation is better.